<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="152988" started-at="2021-08-18T12:31:47 GMT+05:30" finished-at="2021-08-18T12:34:20 GMT+05:30">
    <groups>
    </groups>
    <test name="Case3" duration-ms="148603" started-at="2021-08-18T12:31:47 GMT+05:30" finished-at="2021-08-18T12:34:16 GMT+05:30">
      <class name="TestCases.Case3Success">
        <test-method status="PASS" signature="ini()[pri:0, instance:TestCases.Case3Success@32464a14]" name="ini" is-config="true" duration-ms="49442" started-at="2021-08-18T12:31:47 GMT+05:30" finished-at="2021-08-18T12:32:37 GMT+05:30">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ini -->
        <test-method status="PASS" signature="Success()[pri:0, instance:TestCases.Case3Success@32464a14]" name="Success" duration-ms="60435" started-at="2021-08-18T12:32:37 GMT+05:30" finished-at="2021-08-18T12:33:37 GMT+05:30">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Success -->
        <test-method status="PASS" signature="end()[pri:0, instance:TestCases.Case3Success@32464a14]" name="end" is-config="true" duration-ms="37825" started-at="2021-08-18T12:33:38 GMT+05:30" finished-at="2021-08-18T12:34:16 GMT+05:30">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- end -->
      </class> <!-- TestCases.Case3Success -->
    </test> <!-- Case3 -->
    <test name="Case1" duration-ms="152988" started-at="2021-08-18T12:31:47 GMT+05:30" finished-at="2021-08-18T12:34:20 GMT+05:30">
      <class name="TestCases.Case1">
        <test-method status="PASS" signature="ini()[pri:0, instance:TestCases.Case1@26275bef]" name="ini" is-config="true" duration-ms="108540" started-at="2021-08-18T12:31:47 GMT+05:30" finished-at="2021-08-18T12:33:36 GMT+05:30">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ini -->
        <test-method status="FAIL" signature="Data(java.lang.String,java.lang.String)[pri:0, instance:TestCases.Case1@26275bef]" name="Data" duration-ms="18441" started-at="2021-08-18T12:33:36 GMT+05:30" data-provider="getData" finished-at="2021-08-18T12:33:55 GMT+05:30">
          <params>
            <param index="0">
              <value>
                <![CDATA[abc@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[12345]]>
              </value>
            </param>
          </params>
          <exception class="org.openqa.selenium.ElementClickInterceptedException">
            <message>
              <![CDATA[Element <span> is not clickable at point (1335.6167297363281,16.133331298828125) because another element <div class="listbuilder-popup-content"> obscures it
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'RAKSHANA_RADHAK', ip: '192.168.1.7', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_291'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 64.0.2, javascriptEnabled: true, moz:accessibilityChecks: false, moz:geckodriverVersion: 0.29.1, moz:headless: false, moz:processID: 5408, moz:profile: C:\Users\Admin\AppData\Loca..., moz:shutdownTimeout: 60000, moz:useNonSpecCompliantPointerOrigin: false, moz:webdriverClick: true, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, platformVersion: 10.0, rotatable: false, setWindowRect: true, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: 1db470e3-808f-4569-ad4d-8b2ea861518b]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.ElementClickInterceptedException: Element <span> is not clickable at point (1335.6167297363281,16.133331298828125) because another element <div class="listbuilder-popup-content"> obscures it
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'RAKSHANA_RADHAK', ip: '192.168.1.7', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_291'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 64.0.2, javascriptEnabled: true, moz:accessibilityChecks: false, moz:geckodriverVersion: 0.29.1, moz:headless: false, moz:processID: 5408, moz:profile: C:\Users\Admin\AppData\Loca..., moz:shutdownTimeout: 60000, moz:useNonSpecCompliantPointerOrigin: false, moz:webdriverClick: true, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, platformVersion: 10.0, rotatable: false, setWindowRect: true, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: 1db470e3-808f-4569-ad4d-8b2ea861518b
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:285)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:84)
at TestCases.Case1.Data(Case1.java:36)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.ElementClickInterceptedException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Data -->
        <test-method status="PASS" signature="Data(java.lang.String,java.lang.String)[pri:0, instance:TestCases.Case1@26275bef]" name="Data" duration-ms="19009" started-at="2021-08-18T12:33:56 GMT+05:30" data-provider="getData" finished-at="2021-08-18T12:34:15 GMT+05:30">
          <params>
            <param index="0">
              <value>
                <![CDATA[def@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[45678]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Data -->
        <test-method status="PASS" signature="end()[pri:0, instance:TestCases.Case1@26275bef]" name="end" is-config="true" duration-ms="4916" started-at="2021-08-18T12:34:15 GMT+05:30" finished-at="2021-08-18T12:34:20 GMT+05:30">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- end -->
      </class> <!-- TestCases.Case1 -->
    </test> <!-- Case1 -->
    <test name="Case2" duration-ms="148602" started-at="2021-08-18T12:31:47 GMT+05:30" finished-at="2021-08-18T12:34:16 GMT+05:30">
      <class name="TestCases.Case2Title">
        <test-method status="PASS" signature="ini()[pri:0, instance:TestCases.Case2Title@61386958]" name="ini" is-config="true" duration-ms="50573" started-at="2021-08-18T12:31:47 GMT+05:30" finished-at="2021-08-18T12:32:38 GMT+05:30">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ini -->
        <test-method status="FAIL" signature="FeaturedCourse()[pri:0, instance:TestCases.Case2Title@61386958]" name="FeaturedCourse" duration-ms="60089" started-at="2021-08-18T12:32:38 GMT+05:30" finished-at="2021-08-18T12:33:38 GMT+05:30">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [FEATURED123COURSES] but found [FEATURED COURSES]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [FEATURED123COURSES] but found [FEATURED COURSES]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at TestCases.Case2Title.FeaturedCourse(Case2Title.java:30)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- FeaturedCourse -->
        <test-method status="PASS" signature="end()[pri:0, instance:TestCases.Case2Title@61386958]" name="end" is-config="true" duration-ms="24717" started-at="2021-08-18T12:33:51 GMT+05:30" finished-at="2021-08-18T12:34:16 GMT+05:30">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- end -->
      </class> <!-- TestCases.Case2Title -->
    </test> <!-- Case2 -->
  </suite> <!-- Suite -->
</testng-results>
